### DISCUSSION TITLE
API remot Upload

### DISCUSSION THREAD

**Discussion by horasjey** [prasathmani/tinyfilemanager#882] [Ideas]
plase sir, add remot upload from url :-)

**Comment by prasathmani** [comment]
already option available to upload remote files using public url
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/11345598/201894437-2f44f5c3-127d-4b55-9914-7fe122fd1b4e.png">

**Reply by horasjey** [reply]
Can post crul?
